src/aurelia-slickgrid/services/sort.service.ts

```diff
@@ -302,9 +334,25 @@ export class SortService {
 
       grid.invalidate();
       grid.render();
+
+      if (emitSortChanged) {
+        this.emitSortChanged(EmitterType.local, sortColumns.map(col => {
+          return {
+            columnId: col.sortCol && col.sortCol.id || 'id',
+            direction: col.sortAsc ? 'ASC' : 'DESC'
```

Comment:
change it to 
`direction: col.sortAsc ? SortDirection.ASC : SortDirection.DESC`